packages/backend-data/src/convert_js_resolvers.ts

```diff
@@ -71,12 +78,15 @@ export const convertJsResolverDefinition = (
 
     const resolverName = `Resolver_${resolver.typeName}_${resolver.fieldName}`;
 
+    const amplifyApiEnvironmentName =
+      scope.node.tryGetContext('amplifyApiEnvironmentName') ?? 'NONE';
```

Comment:
I thought we didn't want to change the context parameter, is this intended?

Reply:
Oops, hasty find and replace.